sconstruct

```diff
@@ -545,3 +545,65 @@ env.Alias('appx',[installed_appx_storeSubmission,installed_appx_sideLoadable])
 env.Default(dist)
 
 env.SConscript("tests/sconscript", exports=["env", "sourceDir", "pot"])
+
+
+def generate_module_list(target, source, env):
+	""" 
+	Generate a list of Python modules from compiled '.pyc' files within a specified zipfile.
+
+    This function lists all '.pyc' files contained in a given zipfile, converts their paths to Python module
+    format, performs a sanity check to ensure a specific module is present, and then writes the list of
+    modules to a specified output file for packaging as an AppVeyor artefact.
+
+    Args:

```

Comment:
We usually follow Sphinx style for doc strings see [the section about docstrings in our coding standards](https://github.com/nvaccess/nvda/blob/master/projectDocs/dev/codingStandards.md#docstrings) here.